From e4b3239c16cc7d26719d4f55dfa64a631f416a20 Mon Sep 17 00:00:00 2001
From: Ben Rady <benrady@gmail.com>
Date: Mon, 14 Sep 2015 16:48:51 -0500
Subject: [PATCH 1018/1031] double quotes

---
 public/app.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/public/app.js b/public/app.js
index 9d6edec..e64c607 100644
--- a/public/app.js
+++ b/public/app.js
@@ -93,7 +93,7 @@ learnjs.landingView = function() {
 learnjs.showView = function(hash) { 
   // START: routes
   var routes = {
-    "#problem": learnjs.problemView,
+    '#problem': learnjs.problemView,
     '': learnjs.landingView
   };
   // END: routes
-- 
2.3.8 (Apple Git-58)

